package.json

```diff
@@ -125,16 +123,10 @@
         "re-resizable": "~6.5.5",
         "react": "^16.8.4",
         "react-dom": "^16.8.4",
-        "react-is": "^18.1.0",
-        "react-syntax-highlighter": "^15.5.0",
         "react-toggle": "^4.1.2",
-        "strip-ansi": "^5.2.0",
         "styled-components": "5.2.1",
-        "tar": "^4.4.18",
-        "trim": "^0.0.3",
```

Comment:
Looks like we have them in resolutions as well as dependencies, remove them
Guess it was a way to force things to work